### PR TITLE
Jetpack CP: Update Networking and Storage with adminURL

### PR DESCRIPTION
**Pull request by itsmeichigo**
Part of #5365 

### Description

This PR updates Site models in Networking and Storage with new property / attribute `adminURL`, which will be used to navigate user to WPAdmin page.

To get `adminURL` in the site list response, a new param `admin_url` is added to `options` list in the `/me/sites` request.

The 700+ line change is due to the new model version.

### Testing instructions

Build the app and make sure that there's no crash. Site list should be loaded successfully as before.

### Screenshots

N/A

---
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: Networking/Networking/Model/Site.swift [hunk 1]
```diff
@@ -21,6 +21,10 @@ public struct Site: Decodable, Equatable, GeneratedFakeable, GeneratedCopiable {
     ///
     public let url: String
 
+    /// Site's admin URL.
+    ///
+    public let adminURL: String
```
Comment (pmusolino) [inline Networking/Networking/Model/Site.swift:26]: This will be useful also to me for the Hub Menu project 💯 thank you!

**Review by pmusolino — APPROVED** [review]
Looks good to me. 
Everything seems to look fine and I didn't encounter any crash or issue.

Just a nit:
it would be ideal to add the new model also under `test_it_can_load_the_expected_model_versions`? (Including the previous models)

**Comment by itsmeichigo** [comment]
@pmusolino I just realized that Jaclyn already created a new model version for this app version, so I just made some changes to update model version 59 and removed 60. This caused quite a few changes to the project.pbxproj file, but everything seems to still work properly. Can you please take another look, thanks!

**Review by pmusolino — APPROVED** [review]
LGTM :shipit:

**Pull request merged**